draft-kazuho-httpbis-priority.md

```diff
@@ -137,94 +137,32 @@ multiple experiments from independent research have shown that simpler schemes
-information from the client is of the most value.
-
-Once the negotiation is complete, endpoints MAY stop sending hop-by-hop
-prioritization signals that were not negotiated in order to conserve bandwidth.
-However, endpoints SHOULD continue sending end-to-end signals (e.g., the
-Priority header field), as that might have meaningful effect to other nodes that
-handle the HTTP message.
-
-## Defined Prioritization Scheme Values
-
-This document defines two prioritization scheme values for use with the
-SETTINGS_PRIORITIES setting.
-
-### H2_TREE {#settings-h2-scheme}
-
-This document defines the priority scheme identifier H2_TREE (8-bit value of 1)
-that indicates support for HTTP/2-style priorities ({{!RFC7540}}, Section 5.3).
-
-The H2_TREE priority scheme identifier MUST NOT be be sent in an HTTP/3 settings
-because there is no defined mapping of this scheme. Endpoints MUST treat receipt
-of H2_TREE as a connection error of type H3_SETTINGS_ERROR.
-
-### URGENCY {#settings-this-scheme}
-
-This document defines the priority scheme identifier URGENCY (8-bit value of 2)
-that indicates support for the extensible priority scheme defined in the present
-document.
-
-An intermediary connecting to a backend server SHOULD declare support for the
-extensible priority scheme when and only when all the requests that are to be
-sent on that backend connection originates from one client-side connection that
-has negotiated the use of the extensible priority scheme (see {{fairness}}).
+## Disabling HTTP/2 Priorities
+
+The problems and insights set out above are motivation for allowing endpoints to
+opt out of using the HTTP/2 priority scheme, in favor of using an alternative
+such as the scheme defined in this specification. Endpoints would benefit from
+understanding their peer's intention, so the new
+SETTINGS_DEPRECATE_HTTP2_PRIORITIES is defined. The value of the parameter MUST
+be 0 or 1.
+
+Enpoints MUST send this SETTINGS parameter as part of the first SETTINGS frame.
+When the peer receives the first SETTINGS frame, it learns if the sender has
+deprecated the HTTP/2 priority scheme, by consulting the value of
+SETTINGS_DEPRECATE_HTTP2_PRIORITIES parameter (or through lack of that
+parameter).
+
+Until the client receives the SETTINGS frame from the server, the client SHOULD
+send both the priority signal defined in the HTTP/2 priority scheme (as it sees
+fit) and also that of this prioritization scheme. Once the client learns that
+the HTTP/2 priority scheme is deprecated, it SHOULD stop sending the HTTP/2
+priority signals. If the client learns that the HTTP/2 priority scheme is not
+deprecated, it SHOULD stop sending PRIORITY_UPDATE frames, but MAY continue
+sending the Priority header field, as it is an end-to-end signal that might be
+useful to nodes behind the server that the client is directly connected to.
+
+As the SETTINGS frame precedes any priority signal sent from a client, a server
```

Comment:
This isn't completely true in HTTP/3 if SETTINGS are lost or delayed.

Reply:
This is section is about deprecating _HTTP/2_ priorities, therefore we do not need to worry about H3.

Reply:
Thanks